### PR TITLE
Initial code with unit testing

### PR DESCRIPTION
**Pull request by herpiko**
### Usage
- Edit `conf.json`, change `hostname` value to the dovecot's hostname.
- `HOST=hostname DB=dbname node index.js /path/to/dovecot/log`
- Set the command above as daily cron.
### Test

`npm run test`

### CODE REVIEW

**Comment by emhaye** [comment]
Ok :+1:

File: index.js [hunk 1]
```diff
@@ -0,0 +1,186 @@
+var _ = require("lodash");
+var fs = require("fs");
```
Comment (mdamt) [inline index.js:2]: Let's use ES6 for now.

Comment (herpiko) [inline index.js:2]: const? OK.

**Comment by mdamt** [comment]
ES6, use const and let and strict mode everywhere.

File: index.js (hunk 1 above)
Comment (herpiko) [inline index.js:2]: But this repo supposed to run under node v0.11.3

**Pull request closed without merging**